favorites/templates/favorites/favorites.html

```diff
@@ -0,0 +1,13 @@
+<html>
```

Comment:
The first line of this file should be `<!DOCTYPE html>` for it to be valid, with your `<html>` tag on the second line.